pages/authorDashboard/PKPAuthorDashboardHandler.inc.php

```diff
@@ -297,7 +304,7 @@ function setupTemplate($request) {
 				FORM_TITLE_ABSTRACT,
 				FORM_CITATIONS,
 			],
-			'representationsGridUrl' => $this->_getRepresentationsGridUrl($request, $submission),
+			'representationsGridUrl' => $canAccessProductionStage ? $this->_getRepresentationsGridUrl($request, $submission) : '',
```

Comment:
This will prevent the authorization error from appearing. However, it leaves the Galley tab under Publication with a loading spinner that never resolves.

To address this, I think you'll need to pass `$canAccessProductionStage` to the template itself (see line 346) and then use that to prevent the tab from appearing in `authorDashboard.tpl`:

```html
{if $canAccessProductionStage}
	<tab id="galleys" label="{translate key="submission.layout.galleys"}">
		<div id="representations-grid" ref="representations">
			<spinner></spinner>
		</div>
	</tab>
{/if}
```

The `authorDashboard.tpl` file is in OJS, not pkp-lib, so you're probably going to have to make this template adjustment in OJS, OMP and OPS.

Reply:
Hi @NateWr, thanks for the feed back. I've made these changes and will get them made for OMP and OPS, too.